daemon/response.go

```diff
@@ -251,15 +251,15 @@ func makeErrorResponder(status int) errorResponder {
 // A FileStream ServeHTTP method streams the snap
 type fileStream struct {
 	SnapName string
-	Info     snap.DownloadInfo
+	Info     *snap.Info
```

Comment:
it would make more sense to take an explicit filename and the DownloadInfo than hard coding messaging info in response.go

Reply:
Thank you! I fixed this now.